tests/framework/checks/skips.go

```diff
@@ -225,3 +227,9 @@ func SkipIfARM64(arch string, message string) {
 		ginkgo.Skip("Skip test on arm64: " + message)
 	}
 }
+
+func SkipIfMultusV3() {
+	if strings.ToLower(os.Getenv("KUBEVIRT_WITH_MULTUS_V3")) != "true" {
```

Comment:
Shouldnt it be 
skip if `KUBEVIRT_WITH_MULTUS_V3` == true ?
because it means we dont have V4 which is needed for hotplug right ?